xmake/plugins/project/vstudio/impl/vs201x_vcxproj_filters.lua

```diff
@@ -76,30 +76,32 @@ function _make_filter(filepath, target, vcxprojdir)
             local extraconf = filegroups_extraconf[filegroup] or {}
             local rootdir = extraconf.rootdir
             assert(rootdir, "please set root directory, e.g. add_filegroups(%s, {rootdir = 'xxx'})", filegroup)
-            if not path.is_absolute(rootdir) then
-                rootdir = path.absolute(rootdir, scriptdir)
-            end
-            local fileitem = path.relative(filepath, rootdir)
-            local files = extraconf.files or "**"
-            local mode = extraconf.mode
-            for _, filepattern in ipairs(files) do
-                filepattern = path.pattern(path.absolute(path.join(rootdir, filepattern)))
-                if filepath:match(filepattern) then
-                    if mode == "plain" then
-                        filter = path.normalize(filegroup)
-                        is_plain = true
-                    else
-                        -- file tree mode (default)
-                        if filegroup ~= "" then
-                            filter = path.normalize(path.join(filegroup, path.directory(fileitem)))
+            for _, rootdir in ipairs(table.wrap(rootdir)) do
```

Comment:
If there is more than one rootdir, and the first rootdir already makes the filter have a value. This causes the for-loop to continue until the filter corresponding to the last rootdir is obtained. I think that's a bit of a problem.

Reply:
How do we decide which rootdir filter value to use?

Reply:
I think we can safely stop once a rootdir matches

Reply:
I added a break condition, it ends the loop once it found a matching rootdir